CommunityFridgeMapApi/functions/image/v1/app.py

```diff
@@ -0,0 +1,45 @@
+import json
+import base64
+
+try:
+    from storage import Storage
+except:
+    from dependencies.python.storage import Storage
+
+class ImageHandler:
+    @staticmethod
+    def get_binary_body_from_event(event: dict) -> bytes:
+        """Extract binary data from request body"""
+        assert event["isBase64Encoded"]
+        return base64.b64decode(event["body"])
+
+    @staticmethod
+    def encode_binary_file_for_response(blob: bytes) -> bytes:
+        """
+        Binary response body of lambda functions should be encoded in base64.
+        https://docs.aws.amazon.com/apigateway/latest/developerguide/lambda-proxy-binary-media.html
+        """
+        return base64.b64encode(blob)
+
+    @staticmethod
+    def lambda_handler(event: dict, storage: Storage) -> dict:
+        bucket = "fridge-report"
+        key = storage.write(bucket, "webp", ImageHandler.get_binary_body_from_event(event))
+        url = storage.generate_file_url(bucket, key)
+        return {
+            "statusCode": 200,
```

Comment:
Are there any errors we need to catch? S3 has really high availability, but I wonder if s3 throws any exceptions when an image didn't upload correctly for some reason

Reply:
I think that's a valid concern. Added some error handling in `S3Service` 39f4cf7d5b7f345a36e9cdd2111abf4ebdea471a.
The lambda handler should return 500 on error.